include/cantera/thermo/MixtureFugacityTP.h

```diff
@@ -145,14 +145,11 @@ class MixtureFugacityTP : public ThermoPhase
     virtual void getChemPotentials_RT(doublereal* mu) const;
 
     //! @}
-    /*!
-     * @name  Properties of the Standard State of the Species in the Solution
-     *
-     * Within MixtureFugacityTP, these properties are calculated via a common
-     * routine, _updateStandardStateThermo(), which must be overloaded in
-     * inherited objects. The values are cached within this object, and are not
-     * recalculated unless the temperature or pressure changes.
-     */
+    //! @name  Properties of the Standard State of the Species in the Solution
+    //! Within MixtureFugacityTP, these properties are calculated via a common
+    //! routine, _updateStandardStateThermo(), which must be overloaded in
+    //! inherited objects. The values are cached within this object, and are not
+    //! recalculated unless the temperature or pressure changes.
```

Comment:
At least in some of the cases like this, I think the blank comment line between the heading and the extended description helped with legibility.

Reply:
I added an empty line in all cases.